src/functions-and-arrays.js

```diff
@@ -52,7 +106,24 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray() {
```

Comment:
what's the issue here?

Reply:
Example:
```
function sum() {
return a+b;
}

sum(4,3)
```

I am passing arguments to function when i call it - but my function is not actually expecting to receive anything here :) Which is not good, because there is no way for it to know what is a and b